ui/tests/components/ManagedResources.test.js

```diff
@@ -29,17 +30,18 @@ function shallowSetup() {
 configure({ adapter: new Adapter() });
 describe("ManagedResources", () => {
   describe("renders", () => {
-    it("it should render with correct number of resources", () => {
+    it("it should render with correct number of components", () => {
       const { enzymeWrapper } = shallowSetup();
       expect(enzymeWrapper.find(TextField).length).toEqual(3);
       expect(enzymeWrapper.find(Fab).length).toEqual(3 + 1);
     });
 
     it("it should render with correct number of resources", () => {
       const { enzymeWrapper } = shallowSetup();
-      expect(enzymeWrapper.find({ value: "a" }).length).toEqual(1);
-      expect(enzymeWrapper.find({ value: "b" }).length).toEqual(1);
-      expect(enzymeWrapper.find({ value: "c" }).length).toEqual(1);
+      // + 3 due to each textfield having menu item w resource name
+      expect(enzymeWrapper.find({ value: "a" }).length).toEqual(1 + 3);
+      expect(enzymeWrapper.find({ value: "b" }).length).toEqual(1 + 3);
+      expect(enzymeWrapper.find({ value: "c" }).length).toEqual(1 + 3);
```

Comment:
Trivial: Use const to represent the offset of 3 instead, instead of repeating the value.